dspace/config/clarin-dspace.cfg

```diff
@@ -252,3 +252,8 @@ s3.upload.by.parts.enabled = true
 
 ### The build version is stored in the specific file ###
 build.version.file.path = ${dspace.dir}/config/VERSION_D.txt
+
+
+#### Item View ####
+# Show handle and doi as identifiers - show only DOI if it exists instead of handle by default
+item-page.show-handle-and-doi = false
```

Comment:
why is this in this PR?

Reply:
Because I customized showing the DOI in the Item View page.